services/docker-api-proxy/docker/Caddyfile

```diff
@@ -0,0 +1,11 @@
+:8888 {
+    handle {
+        basicauth {
```

Comment:
So this container has two apps? One reverse-proxy (caddy) that auths, and the actual python app that access docker api. 

Q: does this service need special tuning of resources ( e.g. 1-2 CPUs? )

Reply:
it's not a python application it's `socat` a lightweight binary

No resources are sufficient. Both apps `caddy` and `socat` have a low footprint. If we see the services going to 100% CPU we'll adjust the resource usage